src/lib.rs

```diff
@@ -9,9 +9,11 @@ mod holochain;
 mod router;
 mod routes;
 mod service;
-pub mod tracing;
 pub mod transcode;
 
+#[cfg(any(test, feature = "test-utils"))]
+pub mod tracing;
```

Comment:
Notice, now only available for testing. The library doesn't need to be deciding how people log when consuming it.

Reply:
Renamed the module and function, and updated the module doc comment